assets/css/fonts.css

```
@@ -1,92 +1,43 @@
 @font-face {
-    font-family: GTAmericaExp-Regular;
-    font-weight: 100;
-    font-style: normal;
-    src: url("https://www.expensify.com/font/GT-America-Standard-Light.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Light.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Light.woff2") format("woff2");
-}
-
-@font-face {
-    font-family: GTAmericaExp-Regular;
-    font-weight: 200;
-    font-style: normal;
-    src: url("https://www.expensify.com/font/GT-America-Standard-Thin.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Thin.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Thin.woff2") format("woff2");
-}
-
-@font-face {
-    font-family: GTAmericaExp-Regular;
+    font-family: Expensify-Neue-Regular;
     font-weight: 400;
     font-style: normal;
-    src: url("https://www.expensify.com/font/GT-America-Standard-Regular.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Regular.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Regular.woff2") format("woff2");
-}
-
-@font-face {
-    font-family: GTAmericaExp-Regular;
-    font-weight: 500;
-    font-style: normal;
-    src: url("https://www.expensify.com/font/GT-America-Standard-Medium.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff2") format("woff2");
-}
-
-@font-face {
-    font-family: GTAmericaExp-Regular;
-    font-weight: 600;
-    font-style: normal;
-    src: url("https://www.expensify.com/font/GT-America-Standard-Medium.eot") format("embedded-opentype"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff") format("woff"), url("https://www.expensify.com/font/GT-America-Standard-Medium.woff2") format("woff2");
+    src: url('/fonts/Expensify-Neue-Regular.woff') format('woff'), url('/fonts/Expensify-Neue-Regular.woff2') format('woff2');
```

We should use `woff2` first, then fall back on `woff` only if `woff2` isn't supported:

```suggestion
    src: url('/fonts/Expensify-Neue-Regular.woff2') format('woff2'), url('/fonts/Expensify-Neue-Regular.woff') format('woff');
```